dotnet-desktop-guide/net/wpf/properties/dependency-property-value-precedence.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Dependency property value precedence"
+description: Learn how the precedence of different property-based inputs within the WPF property system determines the effective value of a dependency property.
+ms.date: "09/30/2021"
+dev_langs: 
+  - "csharp"
+  - "vb"
+helpviewer_keywords: 
+  - "dependency properties [WPF], classes as owners"
+  - "dependency properties [WPF], metadata"
+  - "classes [WPF], owners of dependency properties"
+  - "metadata [WPF], dependency properties"
+---
+<!-- The acrolinx score was 92 on 10/01/2021-->
+
+# Dependency property value precedence
+
+<a name="introduction"></a>The workings of the Windows Presentation Foundation (WPF) property system affect the value of a dependency property. This article explains how the precedence of different property-based inputs within the WPF property system determines the effective value of a dependency property.
```

Comment:
These bookmark links from the old articles can be removed. We only link to headings unless there is a specific reason to link to something deeper.